install/mysql/glpi-empty.sql

```diff
@@ -21,7 +21,7 @@
 --
 -- GLPI is distributed in the hope that it will be useful,
 -- but WITHOUT ANY WARRANTY; without even the implied warranty of
--- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
```

Comment:
This one breaks the lint process
```suggestion
-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
```

Reply:
yes I saw that too late. I revert it back